tests/evaluation/benchmark/test_benchmark.py

```diff
@@ -0,0 +1,198 @@
+from datetime import datetime
+from typing import Optional
+from uuid import UUID, uuid4
+
+import pytest
+from pydantic import BaseModel
+from pytest import fixture
+from requests import HTTPError, Response
+
+from intelligence_layer.connectors.studio.studio import (
+    GetBenchmarkResponse,
+    StudioClient,
+)
+from intelligence_layer.evaluation.benchmark.studio_benchmark import (
+    StudioBenchmarkRepository,
+    create_aggregation_logic_identifier,
+    create_evaluation_logic_identifier,
+    type_to_schema,
+)
+from tests.evaluation.conftest import (
+    DummyAggregationLogic,
+    DummyEvaluationLogic,
+)
+
+
+@fixture
+def datatset_id() -> str:
+    return "dataset_id"
+
+
+@fixture
+def get_benchmark_response(datatset_id: str) -> GetBenchmarkResponse:
+    eval_logic = DummyEvaluationLogic()
+    evaluation_identifier = create_evaluation_logic_identifier(eval_logic)
+    aggregation_logic = DummyAggregationLogic()
+    aggregation_identifier = create_aggregation_logic_identifier(aggregation_logic)
+    return GetBenchmarkResponse(
+        id="id",
+        project_id=0,
+        dataset_id=datatset_id,
+        name="name",
+        description="description",
+        benchmark_metadata=None,
+        evaluation_logic=evaluation_identifier,
+        aggregation_logic=aggregation_identifier,
+        created_at=datetime.now(),
+        updated_at=None,
+        last_executed_at=None,
+        created_by=None,
+        updated_by=None,
+    )
+
+
+@fixture
+def evaluation_logic() -> DummyEvaluationLogic:
+    return DummyEvaluationLogic()
+
+
+@fixture
+def aggregation_logic() -> DummyAggregationLogic:
+    return DummyAggregationLogic()
+
+
+def test_type_to_schema() -> None:
```

Comment:
Is this not just testing the`TypeAdapter` method of pydantic?

Reply:
Yeah but we wanted to see if the behavior is intended with what we expect